dll/win32/msgina/lang/ro-RO.rc

```diff
@@ -43,9 +43,9 @@ BEGIN
     EDITTEXT IDC_LOGON_PASSWORD, 60, 76, 155, 14, ES_AUTOHSCROLL | ES_PASSWORD
     LTEXT "Vă autentificați la:", IDC_STATIC, 6, 96, 52, 8
     COMBOBOX IDC_LOGON_DOMAIN, 60, 95, 155, 14, CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
-    PUSHBUTTON "Con&firmă", IDOK, 106, 122, 50, 14, BS_DEFPUSHBUTTON
-    PUSHBUTTON "A&nulează", IDCANCEL, 160, 122, 50, 14
-    PUSHBUTTON "&Oprește", IDC_LOGON_SHUTDOWN, 214, 122, 50, 14
+    PUSHBUTTON "&OK", IDOK, 106, 122, 50, 14, BS_DEFPUSHBUTTON
+    PUSHBUTTON "&Revocare", IDCANCEL, 160, 122, 50, 14
+    PUSHBUTTON "&Oprire", IDC_LOGON_SHUTDOWN, 214, 122, 50, 14
```

Comment:
PUSHBUTTON "&OK", IDOK, 106, 122, 50, 14, BS_DEFPUSHBUTTON
has accelerator collision with
PUSHBUTTON "&Oprire", IDC_LOGON_SHUTDOWN, 214, 122, 50, 14
Check you Romanian Windows!

Reply:
Please, recheck the last commit.

Reply:
I do confirm that The collisions are gone now after switching to O&K everywhere. I do still insist on a screenshot. Download a Romanian 2k3 if you don't have it on your current more modern Windows.